Sources/BraintreeDropIn/Public/BraintreeDropIn/BTDropInResult.h

```diff
@@ -54,13 +54,15 @@ typedef NS_ENUM(NSInteger, BTDropInErrorType) {
 @property (nonatomic, strong, nullable) NSString *deviceData;
 
 /**
- * Fetch a `BTDropInResult` with the customer's most recently vaulted payment method.
+ * Fetch a `BTDropInResult` with the customer's vaulted payment method.
+ * The payment method returned is not guaranteed to be the most recently added payment method.
  * If the last payment method selected from Drop-in was Apple Pay, a `BTDropInResult` with
```

Comment:
```suggestion
 * If the vaulted payment method is Apple Pay, a `BTDropInResult` with
```
Nitpick - but this is probably also not necessarily the last payment method selected right?